src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+
+  width: $card-width;
+  margin-bottom: 46px;
+  margin-right: 48px;
+  padding: 32px 0 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__icon {
+    width: 160px;
+    height: 134px;
+
+    margin: 0 19px 36px;
+  }
+
+  &__info {
+    display: flex;
+    flex-direction: column;
+
+    margin: 0 16px;
+  }
```

Comment:
Instead of it, you could just set paddings to `card`

Reply:
@OleksiiBlezniuk I did it but then appeared trouble with icon - it didn't center. But I found that if make it display: block; - then 'margin: 0 auto' would apply to it. So now it works properly. I hope now my code is right.

Reply:
Yep, it's a good solution